### PR TITLE
SKIP UPGRADE TEST init commit

### PR DESCRIPTION
**Pull request by Khuzaima05**
### Description

Migrating secrets-manager to public

issue: https://github.ibm.com/GoldenEye/issues/issues/6572

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- When merging, use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

**Review by shemau — CHANGES_REQUESTED** [review]
This looks good to me. An existing issue with using github rather than registry for a module.

File: README.md [hunk 1]
```diff
@@ -21,64 +21,42 @@ TODO: Replace me with description of the module(s) in this repo
...
-
 ```hcl
-
+provider "ibm" {
+  ibmcloud_api_key     = "XXXXXXXXXXXXXX"
+  region               = "us-south"
+}
+module "secrets_manager" {
+   source                               = "terraform-ibm-modules/secrets-manager/ibm"
+  version                              = "X.X.X"  # Replace "X.X.X" with a release version to lock into a specific release
```
Comment (ocofaigh) [inline README.md:41]: can you fix the indentation here so all equal signs are aligned?

File: examples/basic/variables.tf [hunk 2]
```diff
@@ -1,28 +1,24 @@
-########################################################################################################################
-# Input variables
-########################################################################################################################
-
 variable "ibmcloud_api_key" {
   type        = string
-  description = "The IBM Cloud API Key"
+  description = "The IBM Cloud API token this account authenticates to"
```
Comment (ocofaigh) [inline examples/basic/variables.tf:3]: its not a token, its an API key

File: examples/complete-private/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Complete example with private only instance and KYOK encryption
+
+This examples handles the provisioning of Secrets-Manager instance, the IAM engine configuration in the recently created instance and a context-based restriction (CBR) rule to only allow Secret Manager to be accessible from within the VPC..
+
+Only private service endpoints are enabled, public are disabled. Secrets Manager instances that are private only do not offer a UI management experience.
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```
Comment (ocofaigh) [inline examples/complete-private/README.md:6]: remove the tf docs content from example

File: examples/complete/variables.tf [hunk 4]
```diff
@@ -1,19 +1,24 @@
...
+variable "prefix" {
+  type        = string
+  description = "Prefix for sm instance"
+  default     = "secrets-manager-test"
+}
+variable "sm_service_plan" {
   type        = string
-  description = "Region to provision all resources created by this example"
-  default     = "us-south"
+  description = "Secrets-Manager Trial plan"
```
Comment (ocofaigh) [inline examples/complete/variables.tf:14]: Update this description to: The Secrets Manager service plan to provision

File: examples/complete/version.tf [hunk 5]
```diff
@@ -1,12 +1,10 @@
 terraform {
-  required_version = ">= 1.3.0, <1.6.0"
-
-  # Ensure that there is always 1 example locked into the lowest provider version of the range defined in the main
-  # module's version.tf (usually a basic example), and 1 example that will always use the latest provider version.
+  required_version = ">= v1.0.0, <1.6.0"
   required_providers {
+    # Pin to the lowest provider version of the range defined in the main module to ensure lowest version still works
```
Comment (ocofaigh) [inline examples/complete/version.tf:4]: remove this comment, it is not accurate - its not pinned to lowest

**Review by ocofaigh — CHANGES_REQUESTED** [review]
I left a few minor comments

File: examples/complete-private/main.tf [hunk 6]
```diff
@@ -0,0 +1,74 @@
+module "resource_group" {
+  source = "git::https://github.com/terraform-ibm-modules/terraform-ibm-resource-group.git?ref=v1.1.4"
```
[the remark this answers is not in the document]
Comment (Khuzaima05) [reply examples/complete-private/main.tf:2]: Done

**Review by shemau — APPROVED** [review]
Approved. All comments from myself and Conall resolved.

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**